packages/react-components/package.json

```diff
@@ -42,10 +43,12 @@
     "@internal/northstar-wrapper": "1.5.1-beta.3",
     "copy-to-clipboard": "~3.3.1",
     "html-to-react": "~1.5.0",
+    "lodash": "^4.17.21",
```

Comment:
Do we import this in any files? Just can't find the code piece related to this

Reply:
`InputBoxComponent.tsx` uses it for debounce.

Reply:
Found it! That file was collapsed by github